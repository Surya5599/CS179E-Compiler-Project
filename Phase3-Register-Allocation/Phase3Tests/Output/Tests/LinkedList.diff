--- Phase3Tests/SelfTestCases/LinkedList.out	2011-04-28 14:55:20.000000000 -0700
+++ Phase3Tests/./Output/Tests/LinkedList.out	2021-03-01 15:09:33.441459300 -0800
@@ -1,26 +1,6 @@
 25
-10000000
-39
-25
-10000000
-22
-39
-25
-1
-0
-10000000
-28
-22
-39
-25
-2220000
--555
--555
-28
-22
-25
-33300000
-22
-25
-44440000
-0
+--- Error (ProgramFault) ---
+invalid argument types: (integer1, data address<Heap.1>+0)
+  Main() line 33
+  LL.Start() line 526
+  List.Print() line 426
