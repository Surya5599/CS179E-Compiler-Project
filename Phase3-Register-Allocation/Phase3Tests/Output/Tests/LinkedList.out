25
--- Error (ProgramFault) ---
invalid argument types: (integer1, data address<Heap.1>+0)
  Main() line 33
  LL.Start() line 526
  List.Print() line 426
